lib/LaTeXML/Package/statements.sty.ltxml

```diff
@@ -159,7 +159,7 @@ sub definitionBody {
       $doc->openElement('dc:title');
       $doc->absorb($title);
       $doc->closeElement('dc:title');}
-    $doc->openElement('omdoc:CMP');
+   if ($doc->isCloseable('omdoc:CMP')) { $doc->openElement('omdoc:CMP'); }
```

Comment:
In fact it was mysterious to me too. I don't know why this change also yields the right result. It might have some side effects that I didn't notice. But https://github.com/KWARC/LaTeXML-Plugin-sTeX/pull/59 makes more sense now.